packages/components/src/ProgressBar/ProgressBar.css

```diff
@@ -39,5 +39,6 @@ progress.SteppedProgressBar[value]::-webkit-progress-value {
 .wrapper {
   display: flex;
   flex-direction: row;
```

Comment:
@jjwlee94 this solution works, but did you try setting a `width: 100%` instead of `flex: 1`? It looks like the ["basic" progress bar has a width set](https://github.com/GetJobber/atlantis/pull/1663/files#diff-ce598deea8ae2e01c12ef5159aed41a90fe39d0eea6ad6caae49dfdb090338f3R6) on it and that might solve for this stepped version without potentially introducing side effects around flex grow/shrink/basis

Reply:
This works as well! Fixed in [`8a724dd`](https://github.com/GetJobber/atlantis/pull/1663/commits/8a724dd512c272d8204c7a7c71ab0e179415e4dc).